synapse/tests/test_lib_stormtypes.py

```diff
@@ -1657,6 +1657,23 @@ async def test_storm_lib_list(self):
             with self.raises(s_exc.StormRuntimeError):
                 await core.callStorm('$lib.list().pop()')
 
+            somelist = ["foo", "bar", "baz", "bar"]
+            q = '''
+                $l.remove("bar")
+                $l.remove("newp")
+            '''
+            opts = {'vars': {'l': somelist}}
+            await core.callStorm(q, opts=opts)
+            self.eq(somelist, ["foo", "baz", "bar"])
```

Comment:
The Storm could should `return($l)` rather than relying on the ability to mutate the var here and below.